### ISSUE TITLE
Allow the saving of the entire ZAP session when scanning with zap-full-scan.py in docker image

### ISSUE THREAD

**Issue by claudijd** [zaproxy/zaproxy#3809]
When performing headless scans using ZAPs zap-full-scan.py script, there are cases where I've already done the work of performing the scan, but I may want to load up the scans session in the full ZAP UI for further investigation, verification, and testing.

My ask here is that the zap-full-scan.py script offer an option to dump the zap session to disk.  This would be incredibly handy for cases where we've done a headless scans of hundreds of servers, but we want to just inspect one scan because it's reporting as potential high security issue.

**Comment by psiinon** [comment]
For reference, I think it should be easy to ensure ZAP creates a perminant session, but we'll need to change the script to export it before the docker instance is closed. As the session is several db files in might be best to zip all of the files. It would also be useful to allow ZAP to easily expand and load a zipped session, but thats just a nice to have ;)

**Comment by claudijd** [comment]
I would agree with having a single state file as import/export, else the user has added overhead to save/restore sessions and/or move them around
